### DISCUSSION TITLE
Does Caffeine Cache support @Cacheable(sync = true)?

### DISCUSSION THREAD

**Discussion by chiyongs** [ben-manes/caffeine#952] [Q&A] [answered]
Hello
I have a question that caffeine cache with `@Cacheable` (sync = true) option is working..
does Caffeine Cache support `@Cacheable` (sync = true) ?

**Comment by ben-manes** [comment, marked as answer]
Yep, it looks like Spring [added](https://github.com/spring-projects/spring-framework/issues/13892#issuecomment-453366709) that in 2015 with Caffeine support.
